patches/Mantle/remove_mtlmanagedobjectadapter_h.patch

```diff
@@ -0,0 +1,22 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Mon, 7 Dec 2020 17:34:08 -0800
+Subject: Remove MTLManagedObjectAdapter.h
+
+We are using an outdated version of Mantle which leverages NSConfinementConcurrencyType,
+an enum which has been deprecated with no replacement as of macOS 10.11.
+The actual solution to this problem is to upgrade Mantle, but for now
+we just stop building the offending adapter.
+
+diff --git a/Mantle/Mantle.h b/Mantle/Mantle.h
+index ebd74e7e435ef008ef29e94d406246c1f7b07a12..81abff872bd597ce6d21bb43be4d19ddc7253088 100644
+--- a/Mantle/Mantle.h
++++ b/Mantle/Mantle.h
+@@ -15,7 +15,6 @@ FOUNDATION_EXPORT double MantleVersionNumber;
+ FOUNDATION_EXPORT const unsigned char MantleVersionString[];
+ 
+ #import <Mantle/MTLJSONAdapter.h>
+-#import <Mantle/MTLManagedObjectAdapter.h>
```

Comment:
I'm a little shocked that this still compiles. What was this code doing? What is the effect of removing this code?

Reply:
@nornagon our version of Mantle in Squirrel is _really_ old - this adapter is held in an external repository now and we should handle that upgrade separately but for now this should be fine i think

Reply:
@nornagon `Mantle` is just a bunch of helpers basically, all this does is stop building _one_ of the Mantle helpers which we don't use.  Unfortunately it's incompatible with the new target SDK version.  It's safer to stop building one helper than it is to update Mantle as part of a Chromium upgrade.  We can upgrade mantle and remove this patch later on.

Reply:
Great, thanks.